enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -954,3 +954,67 @@ def test_can_redeem_policy_no_price(self, mock_lms_client):
         assert response.json() == {
             'detail': f'Could not determine price for content_key: {test_content_key}',
         }
+
+    @mock.patch('enterprise_access.apps.api.v1.views.SubsidyAccessPolicyRedeemViewset.subsidy_client')
```

Comment:
My latest commit got rid of this client, you'll want to mock something like this instead:
```python
        mocked_content_data_from_view = {
            "content_uuid": str(uuid4()),
            "content_key": test_content_key,
            "source": "edX",
            "content_price": None,
        }

        with mock.patch(
            'enterprise_access.apps.api.v1.views.subsidy_access_policy.get_and_cache_content_metadata',
            return_value=mocked_content_data_from_view,
        ):
            # ...yer codez
```
Your mock doesn't _have_ to be a context manager, I just liked that style to sort of mentally distinguish the mock of the price fetch in the view vs. the mock of the price fetch down in the `can_redeem()` logic.

Reply:
turns out I didn't even need to mock that function since the code doesn't get that far anyway.